external/importL10n/locales.js

```diff
@@ -27,7 +27,7 @@ const DEFAULT_LOCALE = "en-US";
 const EXCLUDE_LANG_CODES = ["ca-valencia", "ja-JP-mac"];
 
 function normalizeText(s) {
-  return s.replace(/\r\n?/g, "\n").replace(/\uFEFF/g, "");
+  return s.replace(/\r\n?/g, "\n").replaceAll("\uFEFF", "");
```

Comment:
Why don't you replace the first `replace` by a `replaceAll` ?
I guess it doesn't matter that much but at least it's more explicit than a `g` regex.

Reply:
Do you mean simply changing `replace` into `replaceAll`, or something else?

Note that I don't *think* that we can easily replace the regular expression with a string, considering what it does. Unless I'm missing something the replacement currently does the following, which can't be captured with just a string:
 - `\r` -> `\n`
 - `\r\n` -> `\n`

Reply:
`replaceAll` can take a regex but I missed in the docs that it has to have a `g`...
But as said, it'd slightly improve the readability of the code, that said it's an opinion, so do whatever you want with it.

Reply:
Yep it's fine, r=me.
And `replaceAll` throws when the `g` is missing so it could help to avoid to overlook it.